k8s/apiextentions/apiextentions.go

```diff
@@ -58,6 +59,18 @@ func NewForConfig(c *rest.Config) (*Client, error) {
 	}, nil
 }
 
+// NewInstanceFromConfigFile returns new instance of client by using given
+// config file
+func NewInstanceFromConfigFile(config string) (Ops, error) {
+	newInstance := &Client{}
+	err := newInstance.loadClientFromKubeconfig(config)
+	if err != nil {
+		logrus.Errorf("Unable to set new instance: %v", err)
```

Comment:
I would skip the log message, we should avoid adding logs in packages that are used as libraries

Reply:
Fixed.